src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -285,6 +299,25 @@ export const ConfirmSummary = ({
         ]
     }, [buyAsset, buyAssetFeeAsset, sellAsset])
 
+  const maybeWarning = useMemo(() => {
+    if (nativeAssetBridgeWarning)
+      return (
+        <Alert status='info' borderRadius='lg'>
+          <AlertIcon />
+          <Text translation={nativeAssetBridgeWarning} />
+        </Alert>
+      )
+    if (isUnsupportedKeepKeyP2TrTx)
+      return (
+        <Alert status='info' borderRadius='lg'>
+          <AlertIcon />
+          <Text translation='trade.disableChainflipKeepKeyTaprootReceive' />
+        </Alert>
+      )
+
+    return null
+  }, [isUnsupportedKeepKeyP2TrTx, nativeAssetBridgeWarning])
```

Comment:
Preferably blocking: 

This is perhaps not ideal, as if both `nativeAssetBridgeWarning` and `isUnsupportedKeepKeyP2TrTx` are `true` we'll only show one warning.

Reply:
Nice catch ser. Could indeed happen when going from BTC to USDC with Flip.
I believe this is accidentally fine from a flow perspective: if you're not able to continue with the swap, there is no point in showing you the warning div re: needing native asset for the swap (it *will* show up if you switch wallets and are actually able to continue), so probably fine to keep it only one warning div for the time being.